tests/evaluate/logging/test_fluent.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+
+import mlflow
+from mlflow.entities import Metric
+from mlflow.entities.assessment_source import AssessmentSource, AssessmentSourceType
+from mlflow.entities.evaluation_tag import EvaluationTag
+from mlflow.evaluation import Assessment, Evaluation, log_evaluations
+
+from tests.evaluate.logging.utils import get_evaluation
+
+
+@pytest.fixture
+def end_run_at_test_end():
+    try:
+        yield
+    finally:
+        mlflow.end_run()
```

Comment:
`try-finally` can be removed. pytest handles error in tests. Fine to keep it. It's not harmless.

Reply:
Had no idea! Removed try/finally. Thanks for teaching me this :)